### ISSUE TITLE
Symbol resembling the acute accent

### ISSUE THREAD

**Issue by andreabernini** [DCLP/dclpxsltbox#158]
In P.Oxy. 67.4570 fr. 3 ll. 1 and 9 a symbol resembling an acute accent is used for indicating a pause:

Line 1: https://github.com/DCLP/idp.data/blob/dclp/DCLP/68/67830.xml#l130
Comment of the ed.pr. (p. 82): “Unexplained ink in the form of an acute accent above the line after ι in κ̣ύκλω, presumably indicating a pause before the following clause (similar one in 9).

Line 9 (after ὅλ]ας)
https://github.com/DCLP/idp.data/blob/dclp/DCLP/68/67830.xml#l138 

Since the aim is different, I guess we cannot use the usual symbol for the acute.

**Comment by rla2118** [comment]
This sort of semantic insufficiency we're going to face with other
standard symbols (e.g., grave accent used to prevent division of compound
words).  At this stage, I'm reluctant to spend time on rare exceptions, and
sometimes we might want to encode in a way that reflects the function of
the mark (e.g., as punctuation) rather than to imitate it graphically.  In
other words, I would be inclined not to invent new Leiden+ when some
existing symbol might capture the function of the sign.  @HolgerEssler,
what do you think?

**Comment by paregorios** [comment]
Two issues here, if I may:

1. What character (or glyph) should be used in the text.
2. Should there be any associated/encompassing XML/TEI markup to indicate the editors' judgment concerning the semantics and function of that mark at that point in the text.

If it's visually indistinguishable from a standard accent, then I'd lean toward answering question 1 by putting an accent in the XML. If you want to assert that it's doing something other than we would expect in the majority of cases (i.e., being an accent mark), then you'd use markup (e.g., wrapping it in a ```<pc>``` element to indicate that it's a punctuation mark.

The follow-on question for Leiden+ is: do you want/need to provide a way to make such markup interventions in the text.

**Comment by HolgerEssler** [comment]
I agree with Rodney not to introduce new Leiden+ syntax at this stage 
and would follow Tom's solution of using markup directly in the xml to 
indicate special functions (if determined without doubt).

**Comment by rla2118** [comment]
Wrapping in the <pc> element sounds like a good way of dealing with this.
As for the follow-up question, this should probably be on our Leiden+
desiderata list, but I would give it lower priority than other things. Can
you add it @andreabernini?

**Comment by andreabernini** [comment]
By using markup directly in the xml, should I write something like:
hi rend="pause" ?

**Comment by paregorios** [comment]
What I was advocating was something like:

```
<pc type="pause">'</pc>
```

See: http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-pc.html

**Comment by wsalesky** [comment]
@paregorios 
What is the status of this ticket? Has there been a resolution on the desired display, and the underlying XML? Any data available that I can use to test?

**Comment by paregorios** [comment]
per discussion with @rla2118 and @jcowey, the suggestion of ```<pc>``` sounds good, but implementation to be deferred with respect to higher priorities. There ought to be more research done on the meanings and symbols potentially involved in other examples, and that's not nearly as important as other things at this stage.
